Source/Godunov.cpp

```diff
@@ -376,13 +410,37 @@ pc_umeth_3D(
 
   // Final Z flux
   amrex::ParallelFor(zfxbx, [=] AMREX_GPU_DEVICE(int i, int j, int k) noexcept {
-    pc_cmpflx(i, j, k, bclz, bchz, dlz, dhz, qm, qp, flx3, q3, qaux, cdir);
+    pc_cmpflx(
+      i, j, k, bclz, bchz, dlz, dhz, qm, qp, flx[2], qec[2], qaux, cdir);
   });
 
+  // Fix bcnormal boundaries - always use PLM and don't do N+1/2 predictor
+  // because the user specifies conditions at N
+  // bcnormal used for "Hard" (Inflow=1) and "UserBC" (6)
+  for (int idir = 0; idir < 3; idir++) {
+    if (bclo[idir] == Inflow || bclo[idir] == 6) {
```

Comment:
instead of `6`, I think we have a variable to check that? Or maybe we dont...

Reply:
I don't believe we do. The BC defines come from AMReX and they don't have a UserBC: https://github.com/AMReX-Codes/amrex/blob/e7d3d5a4fad6559ada6bfaa011438f9e7e317ca7/Src/Base/AMReX_BC_TYPES.H#L103

We may want to have our own defines like ERF: https://github.com/erf-model/ERF/blob/9ea6df90fc497bde2bc2a2e538480d4792507b9c/Source/IndexDefines.H#L159

Reply:
Added an enum for this.